defaults/main.yml

```diff
@@ -51,5 +51,5 @@ odoo_daemon: "odoo.service"
 odoo_role_demo_data: false
 
 # HTTP server settings
-odoo_role_odoo_http_interface: "0.0.0.0"
-odoo_role_odoo_proxy_mode: false
+odoo_role_odoo_http_interface: "127.0.1.1"
```

Comment:
This should be `127.0.01`, right?

More context https://github.com/coopdevs/odoo-role/pull/58

Reply:
> This should be `127.0.0.1`, right?
> 
> More context #58

yes thankssss